Apps/WebClient/src/ClientApp/package.json

```diff
@@ -81,6 +81,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/vue-fontawesome": "^0.1.9",
     "@types/vue-clickaway": "^2.2.0",
+    "@types/winston": "^2.4.4",
```

Comment:
We should not use this old version. Can you please remove this line? We are currently having " "winston": "^3.3.3""

Reply:
2.4.4 is a sufficient version to support winston ver 3.3.3. Since it is just the types declaration, it only needs to be updated when new types are introduced and this version is perfectly fine - @types/winston 3.3.3 does not even exist

Reply:
Does the winston 3.3.3 version of the library already include the types?

Reply:
Perhaps. Could remove it and declare it in our declarations file

Reply:
yes, the 3.3.3 is working fine.